e2e/benchmarks/local-benchmark/index.html

```diff
@@ -256,8 +246,13 @@ <h2>TensorFlow.js Model Benchmark</h2>
             tf.env().set('CANVAS2D_WILL_READ_FREQUENTLY_FOR_GPU', true);
           }
 
-          predictionData = await predictAndGetPredictionData(predict, model, inferenceInput, debug);
-
+          const predictionResult = await predictAndGetPredictionData(predict, model, inferenceInput, debug, state.backend);
+          predictionData = predictionResult['data'];;
+          if (debug) {
+            const predictionJsonObject = predictionResult['intermediateData'];
+            const referenceJsonObject = referenceResult['intermediateData'];
+            compareJsons(predictionJsonObject, state.backend, referenceJsonObject, referenceBackend);
```

Comment:
`compareJsons` -> `expectIntermediateObjectsClose`?

Reply:
I rename this  to :  compareAndDownloadErrorValue

This can be used to compare general objects and downloads error values. Not limited to IntermediateObjects